docs/psoc6/quickref.rst

```diff
@@ -738,3 +738,68 @@ Constructor
     num_read = audio_in.readinto(buf)# fill buffer with audio samples from I2S device
 
 
+UART
+----
+
+See :ref:`machine.UART <machine.UART>`.
+
+The following specialization applies to this port:
+
+Constructor
+^^^^^^^^^^^^
+
+.. class:: UART(id)
+
+   Construct a UART object of the given id:
+
+   - ``id`` takes any integer (This value is ignored for PSoC6 port)
+
+Methods
+^^^^^^^
+
+.. method:: UART.init(baudrate, bits, parity, stop, tx, rx, rts, cts, timeout,flow)
+
+    Initialize the UART with the given parameters.
+
+    Mandatory parameters:
```

Comment:
Some arguments are keyword arguments and some are not. Do we not document this here?